packages/data-context/src/sources/HtmlDataSource.ts

```diff
@@ -6,6 +6,7 @@
 import type { DataContext } from '../DataContext'
 import { getPathToDist, resolveFromPackages } from '@packages/resolve-dist'
 import _ from 'lodash'
+import configDefaults from '../../../app/config-defaults'
```

Comment:
It seems like these are pulling the `config-defaults` we use for testing cypress-in-cypress, which is different than the normal config defaults.

Should we just use the cached `namespace` & `socketIoRoute` from the last known "good" config, since we presumably need to have that for the browser to have been launched in the first place? I think `fieldsFromLegacyCfg` should have that?

```
cfg.namespace = fieldsFromLegacyCfg.namespace
cfg.socketIoRoute = fieldsFromLegacyCfg.socketIoRoute
```

Reply:
@tgriesser I've made the changes we discussed and one or two small tweaks.